templates/test/result.html.ep

```diff
@@ -44,6 +44,14 @@
                     % } else
                     % {
                         State:<%= $job->state %>
+                        % if (scalar($job->gru_dependencies) > 0) {
+                            % my $grutext = '';
+                            % for my $grudep ($job->gru_dependencies) {
+                                % $grutext .= join(",", $grudep->gru_task->id, $grudep->gru_task->taskname);
+                                % $grutext .= " ";
+                            % }
+                            waiting for GRU tasks: <%= $grutext %>
```

Comment:
I kinda went back and forth on the wording. 'GRU tasks' means less to someone who doesn't know what a GRU task is, but more to someone who does, especially since we're printing the exact task numbers. Maybe "waiting for background (GRU) tasks"? also, what's the canonical capitalization of 'gru'? :)